README.md

```diff
@@ -371,24 +283,6 @@ In this example we increase the `--line-length` value and also include python (`
 files for formatting - this effectively runs `black` on them. `snakefmt` will also pass
 on the `[tool.black]` settings, internally, to `black`.
 
-## Design
-
-### Syntax
-
-`snakefmt`'s parser will spot syntax errors in your snakefiles:
-
-* Unrecognised keywords
-* Duplicate keywords
-* Invalid parameters
-* Invalid python code
-
-But `snakefmt` not complaining does not guarantee your file is entirely error-free.
-
-### Formatting
-
-Python code is `black`ed.
-
-Snakemake-specific syntax is formatted following the same principles: see [PEP8][PEP8].
```

Comment:
Moving this text to another file creates more complexity than it solves from a docs point of view. If you are determined to remove this from the README, at least add a link to the new document.

Reply:
Happy deleting this then